Content.Client/PDA/PdaBoundUserInterface.cs

```diff
@@ -4,18 +4,20 @@
 using Content.Shared.PDA;
 using JetBrains.Annotations;
 using Robust.Client.UserInterface;
-using Robust.Shared.Configuration;
 
 namespace Content.Client.PDA
 {
     [UsedImplicitly]
     public sealed class PdaBoundUserInterface : CartridgeLoaderBoundUserInterface
     {
+        private readonly PdaSystem _pdaSystem;
+
         [ViewVariables]
         private PdaMenu? _menu;
 
         public PdaBoundUserInterface(EntityUid owner, Enum uiKey) : base(owner, uiKey)
         {
+            _pdaSystem = EntMan.System<PdaSystem>();
         }
 
         protected override void Open()
```

Comment:
i think this should be if (_menu == null) { create it } _menu.OpenCenteredLeft();

it fixed similar issues in other things so might fix this as well

Reply:
Do you have any examples?

Reply:
salv magnet

Reply:
just use the ??= operator